flocker/volume/_ipc.py

```diff
@@ -132,7 +132,7 @@ def receive(self, volume):
         return self._destination.run([b"flocker-volume",
                                       b"--config", self._config_path.path,
                                       b"receive",
-                                      volume.uuid.encode(b"ascii"),
+                                      volume.node_id.encode(b"ascii"),
```

Comment:
just a note(this is pre-existing, clearly) - `...encode(b"ascii")` is a hypercorrection (should just be `"ascii"`).